packages/suite/src/support/messages.ts

```diff
@@ -7980,4 +7980,16 @@ export default defineMessages({
         id: 'TR_MOBILE_APP_PROMO_TEXT_FOOTER',
         defaultMessage: 'Sync & track on your phone with <b>Trezor Suite Lite</b>',
     },
+    TR_CANDIDATE_TRANSACTION_HEADER: {
+        id: 'TR_CANDIDATE_TRANSACTION_HEADER',
+        defaultMessage: 'Candidate Transactions',
+    },
+    TR_CANDIDATE_TRANSACTION: {
+        id: 'TR_CANDIDATE_TRANSACTION',
+        defaultMessage: 'CoinJoin Candidate',
```

Comment:
nitpick, can be fixed in Crowdin, but since there is unfinished TooltipSymbol, would be nice to fix right away
```suggestion
        defaultMessage: 'Coinjoin Candidate',
```

Reply:
add418ab4